Applications/MaterialManager/Client/MaterialManagerClientMaterialBoards.cs

```diff
@@ -1,24 +1,57 @@
-﻿using HomagConnect.Base.Contracts.Enumerations;
-using HomagConnect.Base.Extensions;
-using HomagConnect.Base.Services;
-using HomagConnect.MaterialManager.Contracts.Material.Boards;
-using HomagConnect.MaterialManager.Contracts.Material.Boards.Interfaces;
-using HomagConnect.MaterialManager.Contracts.Statistics;
-using System;
+﻿using System;
 using System.Collections.Generic;
 using System.Globalization;
 using System.Linq;
 using System.Net.Http;
 using System.Text;
 using System.Threading.Tasks;
 
+using HomagConnect.Base.Contracts.Enumerations;
+using HomagConnect.Base.Extensions;
+using HomagConnect.Base.Services;
+using HomagConnect.MaterialManager.Contracts.Material.Boards;
+using HomagConnect.MaterialManager.Contracts.Material.Boards.Interfaces;
+using HomagConnect.MaterialManager.Contracts.Request;
+using HomagConnect.MaterialManager.Contracts.Statistics;
+
+using Newtonsoft.Json;
+
 namespace HomagConnect.MaterialManager.Client;
 
 /// <summary>
 /// materialManager Client for boards
 /// </summary>
 public class MaterialManagerClientMaterialBoards : ServiceBase, IMaterialManagerClientMaterialBoards
 {
+    #region create
+
+    /// <inheritdoc />
+    public async Task<BoardType> CreateBoardType(MaterialManagerRequestBoardType boardTypeRequest)
+    {
+        if (boardTypeRequest == null)
+        {
+            throw new ArgumentNullException(nameof(boardTypeRequest));
+        }
+
+        ValidateRequiredProperties(boardTypeRequest);
+
+        var payload = JsonConvert.SerializeObject(boardTypeRequest);
+        var content = new StringContent(payload, Encoding.UTF8, "application/json");
```

Comment:
Please extend ServiceBase.PostObject to simplify the code.

Reply:
done